src/modules/fundingManager/oracle/interfaces/IFM_PC_ExternalPrice_Redeeming_v1.sol

```diff
@@ -76,7 +76,10 @@ interface IFM_PC_ExternalPrice_Redeeming_v1 is
     /// @notice	Thrown when a redemption queue execution fails
     error Module__FM_PC_ExternalPrice_Redeeming_QueueExecutionFailed();
 
-    // -------------------------------------------------------------------------
+    /// @notice Thrown when the project treasury address is invalid
+    error Module__FM_PC_ExternalPrice_Redeeming_InvalidProjectTreasury();
+
+    //--------------------------------------------------------------------------
```

Comment:
a space between `//` and the `---` is missing

Reply:
Done